frontend/src/_components/ToolTip.jsx

```diff
@@ -8,14 +8,20 @@ export function ToolTip({
   children,
   placement = 'top',
   trigger = ['hover', 'focus'],
-  delay = { show: 800, hide: 100 },
+  delay = { show: 50, hide: 100 },
```

Comment:
When you change common components. please make sure that the other tooltips child component will not be affected.